src/components/more-blogs-section/more-blogs-section.jsx

```diff
@@ -16,7 +16,7 @@ const MoreBlogsSection = ({ relatedPosts, refMoreFrom, title, category }) => {
         <Link to="" className={classnames("text__heading__three__blueTwo", styles.titleStyle)}>More from about <span>{title}</span>→</Link>
       </div>
       <div className={styles.blogsContainer}>
-        {relatedPosts.slice(0, 3).map((post) => <Card data={post} key={post.frontmatter.title} className={styles.cardStyles} category={category} />)}
+        {relatedPosts.slice(0, 3).map((post) => <Card data={post} key={post.frontmatter.title} className={classnames(styles.cardStyles)} category={category} />)}
```

Comment:
This change shouldn't be needed, but it's ok

Reply:
Done